snakemake/script.py

```diff
@@ -1,1369 +1,1374 @@
-__author__ = "Johannes Köster"
```

Comment:
Mhm, this gigantic change in this file is fishy... What is going on here?

Reply:
Maybe windows linebreaks?

Reply:
Thanks a lot for your explanation.. Sorry, new git user here. I think the problem is I pushed from a Windows machine and did not set hard line break settings. Git forces CRLF and replace LF on Windows by default.

Fixed the line break problem. Hopefully it will work!